Common/TableProducer/CMakeLists.txt

```diff
@@ -112,3 +112,8 @@ o2physics_add_dpl_workflow(mftmchmatchingml
                                           O2::CCDB O2Physics::MLCore
                                           O2::ReconstructionDataFormats
                     COMPONENT_NAME Analysis)
+
+o2physics_add_dpl_workflow(produce-ese-table
+                    SOURCES ProduceEseTable.cxx
```

Comment:
```suggestion
o2physics_add_dpl_workflow(ese-table-producer
                    SOURCES eseTableProducer.cxx
```

Reply:
Should be changed now along with the other one. I had a bit of git issues hence the way I implemented suggestions..

Reply:
Thanks